src/main/scala-2.11/com/linagora/gatling/imap/check/ImapCheckSupport.scala

```diff
@@ -18,10 +20,20 @@ trait ImapCheckSupport {
     }
   }
 
+  def hasNoRecent = hasRecent(0)
+
   def hasFolder(expected: String) =
     ImapSimpleCheck(
       _.folderList.contains(expected),
       resp => s"""Unable to find folder '$expected' in ${resp.folderList.mkString(", ")}""")
 
-  def hasNoRecent = hasRecent(0)
+  def hasUid(expected: Int) =
```

Comment:
could you create a value type for uid, I really don't like to mess with uid/msg once again